CMakeLists.txt

```diff
@@ -204,6 +205,9 @@ install(FILES ${COGCORE_API_HEADERS}
     COMPONENT "development"
 )
 
+join_paths(COG_PKGCONFIG_LIBDIR "\${prefix}" "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
Because this `prefix` variable is the one from the `.pc` file (it is not defined in CMake!), what this line does is prefixing the literal `${prefix}` to the value of the `CMAKE_INSTALL_LIBDIR` variable. For example, using `cmake -DCMAKE_INSTALL_LIBDIR=lib …`, this results in the following inside the `.pc` file:

```
prefix=...
libdir=${prefix}/lib
```

…which is the correct thing to do.